.github/workflows/actions-pipeline.yml

```diff
@@ -0,0 +1,75 @@
+name: Build
+
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: false
+  
+on:
+  push:
+    branches: 
+      - 'main'
+      - 'release/*'
+  pull_request:
+    branches: 
+      - 'main'
+      - 'release/*'
+      - '!/docs/'
+      - '!/*.md'
+
+env:
+  RunPeVerify: true  
+  
+jobs:
+
+  build:
+
+    strategy:
+      matrix:
+        configuration: [Debug, Release]
+        projectloadstyle: [ net35, net40, net46, net6_0, net7_0 ]
+
+    runs-on: windows-latest  # For a list of available runner types, refer to
+                             # https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on
+
+    env:
+      Solution_Name: Open-XML-SDK.sln                         # Replace with your solution name, i.e. MyWpfApp.sln.
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    # Install the .NET Core workload
+    - name: Install .NET Core
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: 6.0.x
+
+    # Add  MSBuild to the PATH: https://github.com/microsoft/setup-msbuild
+    - name: Setup MSBuild.exe
+      uses: microsoft/setup-msbuild@v1.0.2
+
+    # Execute all unit tests in the solution
+    # - name: Execute unit tests
+    #  run: dotnet test
+
+    # Restore the projects
+    - name: Restore the Projects
+      run: msbuild $env:Solution_Name /t:Restore /p:Configuration=$env:Configuration
+      env:
+        Configuration: ${{ matrix.configuration }}
```

Comment:
don't need configuration here and in the line. I kin dof like using the nv block for all these projectloadstyle/configuration/etc

Reply:
@twsouthwick this should be better and has the os platforms in it.